packages/forms/src/Components/Select.php

```diff
@@ -30,6 +30,8 @@ class Select extends Field
 
     protected string | Closure | null $searchPrompt = null;
 
+    protected string | Closure | null $url = null;
```

Comment:
The Tables package has a `CanOpenUrl` trait, might be worth making a similar one for the forms package.